### ISSUE TITLE
computer-programming_lab-functions

### ISSUE THREAD

**Issue by gayatripurigilla** [virtual-labs/computer-programming-responsive-iiith#122]
**description**in the answers section two more questions are being displayed**steps to reach the issue**1)select computer-programming lab2)select functions and click on quiz3)answer the questions and check the score and correct answers**expected result**list of answers for 6 questions**actual result**list of answers for 8 questions**Screenshot**![Screenshot__445_](https://gitlab.com/thefirefist/virtual-labs-internship-pranay/uploads/abb47b242e98088d05c8ff4880fdef67/Screenshot__445_.png)

**Comment by Rajadattu** [comment]
Open this link , copy and run the quiz code. https://github.com/Rajadattu/computer-programming-responsive-iiith/blob/rajdutt/src/lab/exp2/quiz.html

Commit id (link for quizresults): ae46c9c

**Comment by adityagoda24** [comment]
valid fix

**Comment by aakarsh7599** [comment]
valid fix

**Comment by Sumaid** [comment]
@Rajadattu, when deployed, the link goes to a blank page where the correct answers should have been instead.

**Comment by Rajadattu** [comment]
Yes, that was my mistake, please copy the code in this file and save it as quizresults.php
[a.txt](https://github.com/virtual-labs/computer-programming-responsive-iiith/files/2146398/a.txt)

**Comment by Sumaid** [comment]
@Rajadattu Make the changes in your own branch, comment the commit id here and create a merge request.

**Comment by Gowthambhujam** [comment]
i removed 7th and 8th questions from result page.
My commit id: bffed48

**Comment by Gowthambhujam** [comment]
valid fix @Gowthambhujam

**Comment by Yerram-Sai-Rachana** [comment]
Valid fix @Gowthambhujam

**Comment by Pallavi98** [comment]
valid fix @Gowthambhujam
